NitroxServer-Subnautica/GameLogic/Entities/Spawning/CrashFishBootstrapper.cs

```diff
@@ -9,18 +9,18 @@ namespace NitroxServer_Subnautica.GameLogic.Entities.Spawning.EntityBootstrapper
 {
     public class CrashFishBootstrapper : IEntityBootstrapper
     {
-        public void Prepare(Entity entity, Entity parentEntity, DeterministicGenerator deterministicBatchGenerator)
+        public void Prepare(WorldEntity entity, WorldEntity parentEntity, DeterministicGenerator deterministicBatchGenerator)
         {
-            Entity crashFish = SpawnChild(entity, deterministicBatchGenerator, TechType.Crash, "7d307502-46b7-4f86-afb0-65fe8867f893");
+            WorldEntity crashFish = SpawnChild(entity, deterministicBatchGenerator, TechType.Crash, "7d307502-46b7-4f86-afb0-65fe8867f893");
             crashFish.Transform.LocalRotation = new NitroxQuaternion(-0.7071068f, 0, 0, 0.7071068f);
             entity.ChildEntities.Add(crashFish);
         }
 
-        private Entity SpawnChild(Entity parentEntity, DeterministicGenerator deterministicBatchGenerator, TechType techType, string classId)
+        private WorldEntity SpawnChild(WorldEntity parentEntity, DeterministicGenerator deterministicBatchGenerator, TechType techType, string classId)
         {
             NitroxId id = deterministicBatchGenerator.NextId();
 
-            return new Entity(new NitroxVector3(0, 0, 0), new NitroxQuaternion(0, 0, 0, 1), new NitroxVector3(1, 1, 1), techType.ToDto(), parentEntity.Level, classId, true, id, null, parentEntity);
+            return new WorldEntity(new NitroxVector3(0, 0, 0), new NitroxQuaternion(0, 0, 0, 1), new NitroxVector3(1, 1, 1), techType.ToDto(), parentEntity.Level, classId, true, id, parentEntity, false, null);
```

Comment:
We should double check the need of this bootstrapper.
Ninja who is playtester for UWE said they were working on crashfish bugs that we alredt fixed in Nitrox. So it might be useless now.

Reply:
They closed the ticket I made for it as "Won't fix" :drained:

Reply:
oof